integration/resources/templates/combination/function_with_intrinsics_resource_attribute.yaml

```diff
@@ -0,0 +1,27 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Description: A template to test timeout support for implicit APIs.
+
+Parameters:
+  IsDevStack: {Type: String, Default: 'true', AllowedValues: ['true', 'false']}
+Conditions:
+  IsDevStack: !Equals [!Ref IsDevStack, 'true']
+  NotIsDevStack: !Not [Condition: IsDevStack]
+
+Resources:
+  MyLambdaFunction:
+    DeletionPolicy: !If [NotIsDevStack, Retain, Delete]
```

Comment:
Why do we need integ-test it?

Reply:
To make sure template with intrinsics in deletion policy it's actually deployable.